templates/glue/stsb/templates.yaml

```diff
@@ -0,0 +1,72 @@
+dataset: glue
+subset: stsb
+templates:
+  50e3a541-108c-4b26-a423-956562d9b3af: !Template
+    id: 50e3a541-108c-4b26-a423-956562d9b3af
+    jinja: Rank on a scale from {{"0.0"}} to {{"5.0"}} how similar the sentences "{{sentence1}}"
+      and "{{sentence2}}" are. ||| {{label}}
```

Comment:
i wonder if we should not have some round-up on the float... wdyt?
It's not really informative beyond the 2 first decimals and I am not sure it's fair to expect such a precise generation from a model

Reply:
Most STS-B labels are quantized to the nearest 0.2, so for T5 during training we just quantized the training labels to the nearest 0.2. https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/data/preprocessors.py#L816
Still got SoTA. Can we do something like `round(label*5)/5` or even `round(label*10)/10` (if we want it to be more generic) in jinja? I think if the model only sees this precision during training, it will not try to generate silly precision during inference.

Reply:
Looks like I can do ``{{label|round(1)}}`` instead of ``{{label}}``. Do you think we should go all-in for the 0.2 rounding, or just the 1-decimal-place?

Reply:
oh awesome!
yes 0.2 rounding looks good since it worked well enough for t5!